its/plugin/tests/src/test/java/com/sonar/it/php/PHPTest.java

```diff
@@ -64,8 +64,16 @@ public void should_support_multimodule_projects() {
       .setProfile("it-profile");
     Tests.executeBuildWithExpectedWarnings(orchestrator, build);
 
-    assertThat(getMeasureAsInt(MULTIMODULE_PROJET_KEY + ":module1", "files")).isEqualTo(4);
-    assertThat(getMeasureAsInt(MULTIMODULE_PROJET_KEY + ":module2", "files")).isEqualTo(2);
+    String componentKey1 = MULTIMODULE_PROJET_KEY + ":module1";
```

Comment:
Why not simply using JUnit's "assume" to disable the whole test case when running on SQ >= 7.6?

Reply:
As discussed, we'll keep the test, as it still involves the scanner "sonar.modules" properties in sonar.properties